alchemy_mock/mocking.py

```diff
@@ -213,18 +236,29 @@ class UnifiedAlchemyMagicMock(AlchemyMagicMock):
         ...
         MultipleResultsFound: Multiple rows were found for one()
 
+        # .get()
+        >>> s.query('foo').filter(c == 'two').filter(c == 'one').get((1, 1))
+        1
+        >>> s.query('foo').filter(c == 'three').get((3, 3))
+        3
+        >>> s.query('foo').filter(c == 'three').get((1, 1))
```

Comment:
hmmm. @shosca not a fan of `get()` only can query within the subselection. thoughts? in other words, do you think it should honor the filter or just build independent identity map?